kura/distrib/src/main/resources/generic-aarch64/kura_install.sh

```diff
@@ -103,6 +103,12 @@ if [ -d /usr/lib/NetworkManager/conf.d/ ]; then
     fi
 fi
 
+# install dnsmasq default configuration
+if [ -f /etc/default/dnsmasq ]; then
```

Comment:
We can decide how to proceed here. This is a lazy approach: copy the file even if the package is not installed on the system. In that case the presence of the config file is harmless. Otherwise, we have to check if dnsmasq is installed (is the package name different according to the OSs?), and copy the file only if needed.
What do you prefer?